### PR TITLE
Added post about blackfire

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

**Comment by Nyholm** [comment]
@carlos-ea if you have a moment to review this, it would be appreciated.

File: _posts/2020-01-26-installing-blackfire-multiple-servers.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+![Instance example]({{ "/images/posts/blackfire/instance-show.png" | absolute_url }})
+
+When we have logged into the instance, we should update it and configure it with
+the server token related to your Blackfire account. You find the correct command 
+in the section "Configure the Agent" in the [Blackfire installation guide](https://blackfire.io/docs/up-and-running/installation). 
+
+![Login to agent instance]({{ "/images/posts/blackfire/login-to-agent.png" | absolute_url }})
+ 
+{% highlight cli %}
+sudo apt-get uppdate
```
Comment (carlos-ea) [inline _posts/2020-01-26-installing-blackfire-multiple-servers.md]: typo

Comment (Nyholm) [reply _posts/2020-01-26-installing-blackfire-multiple-servers.md]: Ha, the only command I didnt not run myself..

**Comment by Nyholm** [comment]
Thank you for the review, I've updated the PR accordingly.

**Pull request merged**